optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/java/org/optaplanner/workbench/screens/domaineditor/client/widgets/planner/PlannerDataObjectEditorViewImpl.java

```diff
@@ -168,8 +177,18 @@ public boolean isFieldPickerEmpty() {
     }
 
     @Override
-    public void initPlanningSolutionScoreTypeOptions( List<Pair<String, String>> options, String selectedScoreType ) {
-        UIUtil.initList( planningSolutionScoreTypeSelector, options, selectedScoreType, false );
+    public void initPlanningSolutionScoreTypeOptions(List<Pair<String, String>> optionPairs) {
+        for (Pair<String, String> optionPair : optionPairs) {
+            Option option = createOption(optionPair);
+            planningSolutionScoreTypeSelector.add(option);
+        }
+    }
+
+    private Option createOption(final Pair<String, String> optionPair) {
+        Option option = (Option) document.createElement("option");
```

Comment:
Out of interest.... open the editor, close it, open it, close it, open it, close it.... examing the DOM. Are spare ```option``` elements (I don't think there are as ```planningSolutionScoreTypeSelector``` would be removed from the DOM and all it's children.. but whenever I see creation without destruction I like to ask).